### ISSUE TITLE
Is the sign of playoutDoublingAdvantage reversed in v1.4.*?

### ISSUE THREAD

**Issue by kaorahi** [lightvector/KataGo#223]
It seems that the behavior of playoutDoublingAdvantage is reversed between v1.3.5 and v1.4.1. In my intuition, `playoutDoublingAdvantage=-2.00` is aggressive and `=2.00` is defensive in v1.4.1. Is this intentional?

In v1.3.5, scoreMean was optimistic with "playoutDoublingAdvantage=2.00". But it is pessimistic in v1.4.1 as follows.

v1.4.1
~~~~
[playoutDoublingAdvantage=2.00]
$ (echo "fixed_handicap 5\nkata-analyze 10"; sleep 10) | ./katago gtp -model net/kata_192x15.bin.gz -config gtp_example.cfg -override-config "dynamicPlayoutDoublingAdvantageCapPerOppLead=0.00, playoutDoublingAdvantage=2.00, logDir=, logFile=/dev/null" | grep -o 'scoreMean [-0-9.]+' | head -n 5
KataGo v1.4.1
Using OpenCL Device 0: Intel(R) Gen9 HD Graphics NEO (Intel(R) Corporation) OpenCL 2.1 NEO
Loaded tuning parameters from: /home/XXX/.katago/opencltuning/tune6_gpuIntelRGen9HDGraphicsNEO_x19_y19_c192_mv8.txt
Loaded config gtp_example.cfg and/or command-line and query overrides
Loaded model net/kata_192x15.bin.gz
Model name: g170-b15c192-s1672170752-d466197061
GTP ready, beginning main protocol loop
scoreMean -79.4797
scoreMean -77.8226
scoreMean -75.076
scoreMean -79.4797
scoreMean -76.8893

[default]
$ (echo "fixed_handicap 5\nkata-analyze 10"; sleep 10) | ./katago gtp -model net/kata_192x15.bin.gz -config gtp_example.cfg -override-config "logDir=, logFile=/dev/null" | grep -o 'scoreMean [-0-9.]+' | head -n 5
...
scoreMean -60.3395
...

[playoutDoublingAdvantage=-2.00]
$ (echo "fixed_handicap 5\nkata-analyze 10"; sleep 10) | ./katago gtp -model net/kata_192x15.bin.gz -config gtp_example.cfg -override-config "dynamicPlayoutDoublingAdvantageCapPerOppLead=0.00, playoutDoublingAdvantage=-2.00, logDir=, logFile=/dev/null" | grep -o 'scoreMean [-0-9.]+' | head -n 5
...
scoreMean -42.2828
...
~~~~

v1.3.5
~~~~
[playoutDoublingAdvantage=2.00]
$ (echo "fixed_handicap 5\nkata-analyze 10"; sleep 10) | ./katago-1.3.5 gtp -model net/kata_192x15.bin.gz -config gtp_example.cfg -override-config "dynamicPlayoutDoublingAdvantageCapPerOppLead=0.00, playoutDoublingAdvantage=2.00, logDir=, logFile=/dev/null" | grep -o 'scoreMean [-0-9.]+' | head -n 5
KataGo v1.3.5
...
scoreMean -42.0346
...

[default]
$ (echo "fixed_handicap 5\nkata-analyze 10"; sleep 10) | ./katago-1.3.5 gtp -model net/kata_192x15.bin.gz -config gtp_example.cfg -override-config "logDir=, logFile=/dev/null" | grep -o 'scoreMean [-0-9.]+' | head -n 5
...
scoreMean -60.0011
...

[playoutDoublingAdvantage=-2.00]
$ (echo "fixed_handicap 5\nkata-analyze 10"; sleep 10) | ./katago-1.3.5 gtp -model net/kata_192x15.bin.gz -config gtp_example.cfg -override-config "dynamicPlayoutDoublingAdvantageCapPerOppLead=0.00, playoutDoublingAdvantage=-2.00, logDir=, logFile=/dev/null" | grep -o 'scoreMean [-0-9.]+' | head -n 5
...
scoreMean -79.4797
...
~~~~

**Comment by lightvector** [comment]
Thanks. Will fix soon and release. There is actually a much bigger bug besides this too that I discovered in the process of fixing this one, still in the case where using PDA + pondering together, will fix that too.

**Comment by lightvector** [comment]
This should be fixed in v1.4.2. Please let me know if you observe otherwise. Thank you again!
https://github.com/lightvector/KataGo/releases

**Comment by kaorahi** [comment]
v1.4.2 works fine. Thanks!
